src/views/virtualmachines/creation-wizard-new/hooks/useCloneVM.ts

```
@@ -3,78 +3,107 @@ import { useNavigate } from 'react-router';
 
   useEffect(() => {
-    const { name, project: targetNamespace } = getValues(CREATE_VM_FORM_FIELDS_VM_DATA.ROOT);
+    handleCloneRequestPhaseChange({
+      cloneRequest,
+      formValues: getValues(CREATE_VM_FORM_FIELDS_VM_DATA.ROOT),
+      navigate,
+      setError,
+      setIsSubmitting,
+      submittedCloneRequest,
+      setSubmittedCloneRequest,
+      t
```

ESLint

[error] Insert `,`

(prettier/prettier)